### PR TITLE
[DeploymentScripts] Added SystemData support

### PR DESCRIPTION
**Pull request by jorgecotillo**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/deploymentScripts.json [hunk 1]
```diff
@@ -840,6 +840,11 @@
           "readOnly": true,
           "type": "string",
           "description": "Type of this resource."
+        },
+        "systemData": {
```
Comment (majastrz) [inline specification/resources/resource-manager/Microsoft.Resources/preview/2019-10-01-preview/deploymentScripts.json]: >systemData [](start = 9, length = 10)

@jcotillo@microsoft.com do we need to update examples to include this?

File: specification/resources/resource-manager/readme.md [hunk 2]
```diff
@@ -535,6 +535,10 @@ directive:
     suppress: OperationsAPIImplementation
     where: $.paths
     reason: OperationsAPI will come from Resources
+  - from: deploymentScripts.json
+    suppress: R3006 #BodyTopLevelProperties
+    where: $.definitions.AzureResourceBase
+    reason: Currently systemData is not allowed
```
Comment (majastrz) [inline specification/resources/resource-manager/readme.md]: @raych1 `systemData` is a valid top-level property now ([https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources)). Can we file a bug for the linter to stop complaining about that?

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

File: specification/resources/resource-manager/readme.md (hunk 2 above)
Comment (jianyexi) [reply specification/resources/resource-manager/readme.md]: the linter has fixed it in new version, and will be applied soon ,
currently, the places you suppressed are incorrect,
it should be like this :
 - from: deploymentScripts.json
    suppress: R3006 #BodyTopLevelProperties
    where: 
    - $.definitions.DeploymentScript.properties
    - $.definitions.AzureCliScript.properties
    - $.definitions.AzurePowerShellScript.properties
    reason: Currently systemData is not allowed

**Comment by raych1** [comment]
Thanks @jianyexi. @jorgecotillo , can you follow Jianye's comment to correct the suppression in readme?

**Review by jorgecotillo — COMMENTED** [review]
@raych1 all checks passed, updated examples and included the proper suppress information. Please proceed to merge the code if there are no other outstanding comments.

**Comment by jorgecotillo** [comment]
@raych1 all checks passed, updated examples and included the proper suppress information. Please proceed to merge the code if there are no other outstanding comments.

**Review by filizt — APPROVED** [review]
LGTM. Note: We chatted on this PR and the linter error offline. When the linter update is live, we can remove the supression.

**Review by raych1 — APPROVED** [review]

**Pull request merged**